docs/products/kafka/howto/group-requests.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Group requests
+limited: true
+---
+
+The group requests page allows you to view and track ownership requests made by you and other members of your group for Aiven for Apache Kafka resources.
+These requests can include claims for existing topics or requests to create new topics.
+
+### Key elements
+
+The group requests page displays the following key elements:
+
+- **Topic**: Name of the topic
+- **Type**: Indicates the type of request
+- **Status**: Current status of the request
+  - **Pending**: Awaiting approval or decline
+  - **Approved**: Request approved and processed
+  - **Declined**: Request declined, topic available for another group
+  - **Deleted**: Request has been deleted
```

Comment:
We do have additional statuses here that could appear as well as mentioned in the approvals comment.

Reply:
Added them.